client/signup/steps/rewind-migrate/index.jsx

```diff
@@ -0,0 +1,107 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import SignupActions from 'lib/signup/actions';
+import ActivityLogRewindToggle from 'my-sites/stats/activity-log/activity-log-rewind-toggle';
+import { getRewindState } from 'state/selectors';
+
+class RewindMigrate extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+
+		// Connected props
+		siteId: PropTypes.number.isRequired,
+		rewindIsNowActive: PropTypes.bool.isRequired,
+	};
+
+	/**
+	 * Before component updates, check if we have to go somewhere else.
+	 * If Rewind was activated, user clicked the Switch now button so let's go to migrate creds.
+	 *
+	 * @param {object} nextProps Props received by component for next update.
+	 */
+	componentWillUpdate( nextProps ) {
+		if ( this.props.rewindIsNowActive !== nextProps.rewindIsNowActive ) {
+			SignupActions.submitSignupStep( {
+				processingMessage: this.props.translate( 'Migrating your credentials' ),
+				stepName: this.props.stepName,
+			} );
+			this.props.goToNextStep();
+		}
+	}
+
+	stepContent = () => {
+		const { translate, siteId } = this.props;
+
+		return (
+			<div className="rewind-migrate__card rewind-switch__card">
+				<Card className="rewind-migrate__content rewind-switch__content">
+					<h3 className="rewind-migrate__title rewind-switch__heading">
+						{ translate( 'Migrate credentials' ) }
+					</h3>
+					<img src="/calypso/images/illustrations/almost-there.svg" alt="" />
```

Comment:
Should we add the alt text here?

Reply:
Decorative images are not required to have a text. They can just have an empty alt attribute.